<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin_MainWindow</class>
 <widget class="QMainWindow" name="admin_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1328</width>
    <height>953</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>欢迎登陆大麦场！</string>
  </property>
  <property name="windowIcon">
   <iconset resource="mysource.qrc">
    <normaloff>:/images/物业管理.png</normaloff>:/images/物业管理.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QFrame" name="frame_top">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>1306</width>
      <height>75</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>75</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>75</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">#frame_top{border:2px solid rgb(121,121,121)}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>740</width>
       <height>56</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_top">
      <item>
       <widget class="QLabel" name="label_title">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>531</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1720</width>
          <height>1030</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>28</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignRight|Qt::AlignTop">
       <widget class="QPushButton" name="pushButton_reback">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">selection-background-color: rgb(170, 85, 0);</string>
        </property>
        <property name="text">
         <string>注销</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_left" native="true">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>93</y>
      <width>224</width>
      <height>781</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>224</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">#widget_left{border:2px solid rgb(121,121,121)}</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>202</width>
       <height>756</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_left_2" stretch="0,0,0,0,0,0,0,0,0,0,0,0,0,0">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton_12">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>51</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>菜  单</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_13">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>51</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>用户管理</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_22">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>51</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>用户家属管理</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_14">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>51</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>车辆管理</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_15">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>51</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>楼宇管理</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_23">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>51</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>房间管理</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_16">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>51</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>设备管理</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_17">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>51</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>收费项目管理</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_18">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>51</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>支付管理</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_19">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>51</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>投诉管理</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_24">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>51</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>备份数据库</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_20">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>51</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>登录日志</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_21">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>51</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>小区信息</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_right">
    <property name="geometry">
     <rect>
      <x>888</x>
      <y>93</y>
      <width>429</width>
      <height>797</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>429</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>429</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">#frame_right{border:2px solid rgb(121,121,121)}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>396</width>
       <height>771</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="ver_right_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>141</width>
            <height>51</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>141</width>
            <height>51</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>管理员信息</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="minimumSize">
             <size>
              <width>72</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>72</width>
              <height>31</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Agency FB</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>登录名：</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="minimumSize">
             <size>
              <width>72</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>72</width>
              <height>31</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Agency FB</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>姓    名：</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit_name">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>311</width>
              <height>51</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>311</width>
              <height>51</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>宋体</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_4">
            <property name="minimumSize">
             <size>
              <width>72</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>72</width>
              <height>31</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Agency FB</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>电    话：</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEdit_tel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>311</width>
              <height>51</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>311</width>
              <height>51</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>宋体</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_logname">
            <property name="minimumSize">
             <size>
              <width>311</width>
              <height>51</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>311</width>
              <height>51</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>宋体</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QPushButton" name="pushButton">
          <property name="minimumSize">
           <size>
            <width>131</width>
            <height>51</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>131</width>
            <height>51</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>修 改</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCalendarWidget" name="calendarWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>381</width>
          <height>321</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>430</width>
          <height>320</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_6">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="minimumSize">
           <size>
            <width>91</width>
            <height>51</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>91</width>
            <height>51</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>当前时间：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>290</width>
            <height>51</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>宋体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <zorder>widget_left</zorder>
   <zorder>frame_top</zorder>
   <zorder>frame_right</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1328</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="mysource.qrc"/>
 </resources>
 <connections/>
</ui>
